Gems/Atom/Tools/MaterialCanvas/Assets/MaterialCanvas/TestData/cosh.materialcanvasnode.azasset

```diff
@@ -0,0 +1,57 @@
+{
+    "Type": "JsonSerialization",
+    "Version": 1,
+    "ClassName": "DynamicNodeConfig",
+    "ClassData": {
+        "category": "Math Functions",
+        "title": "cosh",
```

Comment:
Do we see much real-world application of hyperbolic trig functions? I'm not sure if it's worth introducing and polluting your node space. If anything I could see an artist using it unintentionally and getting confused about the results. (maybe an advanced artist with a physics degree is trying to model the shape of a catenary in a shader, but even then we should be doing that with other means).